src/lib/pages/frontsite/PatronProfile/LoansList.js

```diff
@@ -1,37 +1,45 @@
-import { Pagination } from '@components/Pagination';
+import { PatronPagination } from './PatronPagination';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { Container, Item } from 'semantic-ui-react';
+import { Item } from 'semantic-ui-react';
 
 export default class LoansList extends Component {
   render() {
     const {
-      activePage,
-      isLoading,
-      onPageChange,
       loans,
-      rowsPerPage,
       renderListEntry,
       noLoansCmp,
+      patronShowLink,
+      onPageChange,
+      activePage,
+      rowsPerPage,
+      isLoading,
     } = this.props;
+
     return loans.total > 0 ? (
       <>
+        <PatronPagination
+          currentPage={activePage}
+          currentSize={rowsPerPage}
+          loading={isLoading}
+          onPageChange={onPageChange}
+          items={loans}
+        />
+        {patronShowLink && patronShowLink}
```

Comment:
probably should work properly with only
```suggestion
        {patronShowLink}
```